packages/smooth_app/lib/pages/preferences/user_preferences_faq.dart

```diff
@@ -75,7 +75,9 @@ class UserPreferencesFaq extends AbstractUserPreferences {
         body: Column(
           children: <Widget>[
             ListTile(
-              leading: Image.asset('assets/app/smoothie-icon.1200x1200.png'),
+              leading: SvgPicture.asset(
+                'assets/app/release_icon.svg',
```

Comment:
I would use a local `const String` as you're using it twice.

Reply:
Applied in my last commit :)